### ISSUE TITLE
Add a Reply-To header

### ISSUE THREAD

**Issue by julbrs** [muratgozel/MIMEText#45]
Hi,

It seems that we are struggling to add a "Reply-To" header (which is different from the "From" header). What is the best way to define a "Reply-To"? 

```
const message = createMimeMessage();
message.setHeader('Reply-To', contact.email);
```

generate the following message:

```
MIMETEXT_INVALID_HEADER_VALUE: You specified an invalid value for the header Reply-To
```

Thanks!

**Comment by ryan-vaneck** [comment]
I am also seeing this issue

**Comment by yukikywong** [comment]
I also see the same problem with setting the `Sender` header.
```
const message = createMimeMessage();
message.setHeader("Sender", "test@example.com");
```
When setting "Sender", it is validating if the value is an instance of a `Mailbox`. When I try to import `Mailbox` however
```
import { createMimeMessage, Mailbox } from "mimetext";

```
I can only import the `Mailbox interface` but not the class, so I cannot call the `Mailbox` constructor to create a Mailbox.

**Comment by Gowee** [comment]
Inspired by @yukikywong, this idea comes up to me:

`msg.setHeader("Reply-To", createMimeMessage().setSender(sender_address));`

It does work well even though be tricky.

**Comment by muratgozel** [comment]
> Inspired by @yukikywong, this idea comes up to me:
> 
> `message.setHeader("Reply-To", createMimeMessage().setSender("test@example.com"));`
> 
> where `setSender` just returns a `Mailbox` instance.
> 
> It does work well even though be tricky.

yo what? how did you even come up with this, this is smart hahah

sorry for the delay, im taking care of it right now

**Comment by muratgozel** [comment]
hey again, the `Reply-To` header  is specially handled by the lib and proper way to add it is by creating an instance of `Mailbox`: `msg.setHeader('Reply-To', new Mailbox('asd@asd.com'))` as its just become available as of version v3.0.19. you have to import the `Mailbox` class: `import { createMimeMessage, Mailbox } from 'mimetext'`. closing this issue as it should all be resolved, open again if you face any problems.

**Comment by j8jacobs** [comment]
@muratgozel want to verify if multiple reply-to addresses can be specified here?

I've confirmed that the `msg.setHeader('Reply-To', new Mailbox('asd@asd.com'))` is the fix here - but when I try doing with multiple MailBoxes I get the same error: MIMETEXT_INVALID_HEADER_VALUE: The value for the header "Reply-To" is invalid.


For example: `msg.setHeader('Reply-To', [new Mailbox('asd@asd.com'), new Mailbox('asd+2@asd.com')])` won't work for me.

Thanks for a great library btw!
